CHANGELOG.next.asciidoc

```diff
@@ -374,6 +374,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix `google_workspace` pagination. {pull}24668[24668]
 - Fix Cisco ASA parser for message 302022. {issue}24405[24405] {pull}24697[24697]
 - Fix date parsing in GSuite/login fileset. {issue}24694[24694]
+- Add `proxy_url` config for httpjson vs input. {issue}24615[24615] {pull}24662[24662]
```

Comment:
This is under the Bugfixes section, could you please move it under the `Added` one?